packages/types/src/Type.ts

```diff
@@ -172,15 +174,13 @@ export default class Type extends Text {
   private static _removeTraits (): Mapper {
     return (value: string): string => {
       return value
-        // `T :: AccountId` -> `T::AccountId` https://github.com/paritytech/substrate/issues/1244
-        .replace(/\s+::\s+/g, '::')
+        // remove all whitespaces
```

Comment:
The hammer approach :)

Reply:
Hope it works... If there are whitespaces, they should be handled before (e.g. `as HasCompact`)

Reply:
I don't think it will have any negative impact at all. As long as we track the different options, we are fine. (And we still do that)